draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -214,7 +214,7 @@ and the ones in {{lora-alliance-spec}} is as follows:
    application server will do the SCHC C/D and F/R.
 
    o The LPWAN-AAA Server is the LoRaWAN Join Server. Its role is to manage and
-   deliver security keys in a secure way, so devices root key is never exposed.
+   deliver security keys in a secure way, so that devices root keys are never exposed.
```

Comment:
Thank you @dbarthel-ol for this useful PR! As discussed could you please undo this change ? I can not edit the PR

Reply:
Following our discussion, I suggest
"in a secure way, so that the devices root key is never exposed."